### PR TITLE
SW-1786 Make update message always show on 0.11.0rc1

### PR DESCRIPTION
**Pull request by khaledsherkawi**
SW-1786 Make 2-step update message notification always show on 0.11.0rc1

### CODE REVIEW

File: octoprint_mrbeam/static/js/messages.js [hunk 1]
```diff
@@ -241,7 +241,11 @@ $(function () {
                 var lmid = -1;
                 self.messages().forEach(function (myMessage) {
                     lmid = Math.max(lmid, myMessage.id);
-                    if (myMessage.notification && myMessage.id > self.lastMessageId) {
+                    if (myMessage.notification && (myMessage.id > self.lastMessageId
+                    // The below condition is only added to keep the 0.11.0 2-step update message notification showing
+                    // if the user stayed on the 0.11.0rc1 version and didn't execute the 2nd software update
+                    // TODO: Should be removed after 0.11.0 stable release
```
Comment (Yelko-mrbeam) [inline octoprint_mrbeam/static/js/messages.js]: did you create a ticket for it?

File: octoprint_mrbeam/static/js/messages.js [hunk 2]
```diff
@@ -241,7 +241,11 @@ $(function () {
...
+                    || (myMessage.id === 3 && MRBEAM_PLUGIN_VERSION === "0.11.0rc1"))) {
```
Comment (Yelko-mrbeam) [inline octoprint_mrbeam/static/js/messages.js:249]: will the message.id always be 3 for the update message? 🤔

File: octoprint_mrbeam/static/js/messages.js (hunk 1 above)
Comment (khaledsherkawi) [reply octoprint_mrbeam/static/js/messages.js]: Attached

File: octoprint_mrbeam/static/js/messages.js (hunk 2 above)
Comment (khaledsherkawi) [reply octoprint_mrbeam/static/js/messages.js:249]: Yes (but only as solid as it can be) 
Added to the comment

**Review by Yelko-mrbeam — APPROVED** [review]

**Pull request merged**